0.X/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.17.2
+FROM alpine:3.18.0
```

Comment:
The PR request says 3.17.3 in the description and 3.18.0 in the title, I am assuming we just didn't update the description to match the title? The commit even says 3.17.3?

Reply:
As I initially upgraded alpine to 3.17.3, forgot to change it to 3.18. I will change both the commit and description to 3.18. .